src/Confluent.SchemaRegistry/Rest/Authentication/BasicHttpAuthenticator.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2016-2018 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace Confluent.SchemaRegistry
+{
+    /// <summary>
+    ///     A HTTP authenticator implementing the 'Basic' scheme.
+    ///
+    ///     See: https://datatracker.ietf.org/doc/html/rfc7617
+    /// </summary>
+    public class BasicHttpAuthenticator : IHttpAuthenticator
```

Comment:
`BasicaAuthenticaitonHeaderValueProvider` i think. I know it's a mouth full, but it's going to be little used and i think more descriptive wins.